frontend/storybook/src/components/entry/imagelabel/one.ts

```diff
@@ -0,0 +1,38 @@
+import "@elements/entry/admin/images/meta/page";
```

Comment:
The file location here should match the storybook - `entry/admin/images/meta/...`